### ISSUE TITLE
If you are using a module and expect the file to exist on the remote, see the remote_src option

### ISSUE THREAD

**Issue by MarkoS046** [leihs/leihs#851]
I have the same issues her. 

Tried to change it to "TARCMD=`command -v gnutar || command -v tar || command -v gtar` with no success. As well as "./scripts/deploy -s" or "./scripts/deploy --separate".

Can anyone advise me on how to solve this?

Much appreciated!

> There are some packages where it is called `gnutar`, so it would be better to just add `gtar` to this list instead of replacing it.
> Feel free to send a Pull Request if you want this resolved quickly.

_Originally posted by @MarkoSchiefelbein in https://github.com/leihs/leihs/issues/716#issuecomment-645412238_

**Comment by MarkoSchiefelbein** [comment]
Changed it to "command -v gtar || command -v tar" while it was deploying, which worked so far. Some process seems to overwrite the edit at the beginning of deployment.

Anyway, I got a new error:
"An exception occurred during task execution. To see the full traceback, use -vvv. The error was: If you are using a module and expect the file to exist on the remote, see the remote_src option
fatal: [***]: FAILED! => changed=false 
  msg: |-
    Could not find or access '/Users/***/***hosting/leihs/deploy/../mail/target/leihs-mail.jar' on the Ansible Controller.
    If you are using a module and expect the file to exist on the remote, see the remote_src option"

Any idea anyone?
